src/utils/validationUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { z } from 'zod';
+
+export const validateEmailInput = (input: string) => {
+  try {
+    const emailSchema = z.string().email();
+    emailSchema.parse(input);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
+export const validatePasswordInput = (input: string) => {
+  try {
+    const passwordSchema = z.string().min(1);
+    passwordSchema.parse(input);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
+export const validateSerialNumberInput = (input: string) => {
+  try {
+    const SNSchema = z.coerce.number().gt(0);
+    SNSchema.parse(input);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
+const validateCurrencyFormat = (input: string) => {
+  // confirms that any input has either 0 or 2 decimal digits
```

Comment:
I like that you included a comment! Super helpful. I would also just move this to a docstring so it shows up nicely in VSCode

```
/**
 * .. your comment...
 */
```